themes/simple/css/admidio.css

```diff
@@ -609,3 +609,18 @@ tr.admidio-event-highlight {
 #plugin-login-form .btn {
     margin-bottom: 1rem;
 }
+
+
+/***********************************/
+/* Changelog configuration  */
+/***********************************/
+
+div.changelog-tableselect-header {
```

Comment:
Instead of setting this explicitly in the global css file you can set the class `fw-bold` in the template at that place.